lang/csharp/src/apache/test/Avro.test.csproj

```diff
@@ -32,13 +32,22 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="coverlet.collector" Version="3.1.2">
```

Comment:
Please do not hardcode package versions in csproj files. They should be defined in versions.props